corpus/convert.py

```diff
@@ -326,3 +330,69 @@ def run(self):
                     segment.fullname() in segments_whitelist
                 ):
                     f.write(segment.orth + "\n")
+
+
+class CorpusToRawWavHDFJob(Job):
```

Comment:
I did not notice it before, but `RawWav` is not an optimal term. Because if we use `soundfile` instead of `wave` (very good suggestion from @vieting) we can load "any" data. And then we should name the job `CorpusToPcmHDF`, as we are storing PCM data.